front/migrations/db/migration_177.sql

```diff
@@ -1 +1 @@
-DROP TABLE IF EXISTS "data_source_view_for_conversations";
\ No newline at end of file
+DROP TABLE IF EXISTS "data_source_view_for_conversations";
```

Comment:
Rebase error ?

Reply:
It's was my editor configured to add a final empty line on file, but no change on the query. I updated it to remove the diff!